Applications/CausalLM/layers/gpt_oss_moe_layer_cached.cpp

```diff
@@ -378,10 +378,13 @@ void CachedSlimGptOssMoELayer::incremental_forwarding(
 
 // Evict experts
 #pragma omp parallel
-    while (loaded_expert_deque.size() > 16) {
+    while (true) {
       int target_idx;
       {
         std::lock_guard<std::mutex> lock(cache_mutex);
+        if (loaded_expert_deque.size() > 16)
```

Comment:
1. ```>``` --> ```<``` ? or ```<=```?
2. need a comment or properly-named constant for 16. (what does it mean)